src/core/lib/channel/channel_stack.h

```diff
@@ -156,6 +156,10 @@ typedef struct {
   /* Implement grpc_call_get_peer() */
   char *(*get_peer)(grpc_exec_ctx *exec_ctx, grpc_call_element *elem);
 
+  /* Implement grpc_channel_get_info() */
+  void (*get_channel_info)(grpc_exec_ctx *exec_ctx, grpc_channel_element *elem,
```

Comment:
Any reason we're not folding peer in with this? (we may have talked about it and my memory is spotty)